src/deploy-cromwell-on-azure/scripts/helm/templates/cromwell-deployment.yaml

```diff
@@ -50,10 +50,11 @@ spec:
             - mountPath: /cromwell-tmp
               name: cromwell-tmp-claim
 {{- $storageAccount  := .Values.persistence.storageAccount -}}
+{{- $cromwellContainers  := .Values.cromwellContainers -}}
 {{- range .Values.internalContainersMIAuth }}
 {{- $path := printf "/%s/%s" .accountName .containerName }}
 {{- $claimName := printf "%s-%s-claim1" .accountName .containerName }}
-{{- if eq $storageAccount .accountName }}
+{{- if and (eq $storageAccount .accountName) (has .containerName $cromwellContainers)}}
```

Comment:
is this case insensitive?

Reply:
It is not, there is no case insensitive comparison in the helm template language from what I can tell. I converted the strings to lowercase for comparison now though.